src/appleseed.studio/mainwindow/mainwindow.cpp

```diff
@@ -1472,29 +1485,34 @@ namespace
                 AABB2i(
                     Vector2i(x0, y0),
                     Vector2i(x1, y1)));
+            m_attribute_editor->refresh(
+                project.get_frame()->get_parameters());
         }
 
       private:
         const QRect m_rect;
+        const AttributeEditor* m_attribute_editor;
     };
 }
 
 void MainWindow::slot_clear_render_region()
 {
-    m_rendering_manager.schedule(
-        auto_ptr<RenderingManager::IScheduledAction>(
-            new ClearRenderRegionAction()));
+    auto_ptr<RenderingManager::IScheduledAction> clear_render_region_action(
+        new ClearRenderRegionAction(m_attribute_editor));
 
-    m_rendering_manager.reinitialize_rendering();
+    if (m_rendering_manager.is_rendering())
```

Comment:
By the way, you can probably use `RenderingManager::schedule_or_execute()` here.